common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -559,6 +559,41 @@ def test_navigation_buttons(self):
             sequence_ui_events
         )
 
+    def test_accordion_events(self):
+        # navigate to second subsection via accordion
+        self.course_nav.go_to_section('Test Section 1', 'Test Subsection 1,2')
+
+        # navigate to first section of next chapter via accordion
+        self.course_nav.go_to_section('Test Section 2', 'Test Subsection 2,1')
+
+        # test UI events emitted by navigation
+        filter_outline_ui_event = lambda event: event.get('name', '') == 'edx.ui.lms.outline.selected'
+
+        outline_ui_events = self.wait_for_events(event_filter=filter_outline_ui_event, timeout=2)
+
+        #TODO: test current and target url's
```

Comment:
I'm not sure if this will work, but try passing in a predetermined `url_name` as a meta-data field when the `XBlockFixtureDesc` is created in this class' `setUp` method.